source/unified-test-format/tests/valid-pass/expectedEventsForClient-topologyDescriptionChangedEvent.yml

```diff
@@ -0,0 +1,40 @@
+description: "expectedEventsForClient-topologyDescriptionChangedEvent"
+
+schemaVersion: "1.20"
+
+runOnRequirements:
+  - topologies:
+      - replicaset
+    minServerVersion: "4.4" # awaitable hello
+
+tests:
+  - description: "can assert on values of newDescription and previousDescription fields"
+    operations:
+      - name: createEntities
+        object: testRunner
+        arguments:
+          entities:
+            - client:
+                id: &client client
+                uriOptions:
+                  directConnection: true
```

Comment:
Assuming the cluster is launched with mongo-orchestration, wouldn't we expect the connection string to include multiple seeds and a `replicaSet` option?

I expect the multiple seeds would conflict with `directConnection`, as mentioned in [directConnection URI option with multiple seeds or SRV URI](https://github.com/mongodb/specifications/blob/master/source/uri-options/uri-options.rst#directconnection-uri-option-with-multiple-seeds-or-srv-uri).

@ShaneHarvey how did this pass in Python?

Reply:
The python test suite helper to create a MongoClient always connects with a single seed and only adds the replicaSet option if directConnection is not given (or False): 
https://github.com/mongodb/mongo-python-driver/blob/e37394d4029d61f72a02dc465fbfc9afe35a5701/test/utils.py#L570-L579

Reply:
> always connects with a single seed and only adds the replicaSet option if directConnection is not given (or False)

This sounds like something that needs to be clarified in the Unified Test Format spec, when we talk about feeding the test runner with a connection string. I imagine these may be the first unified tests with `directConnection`, so this hasn't come up before.